### DISCUSSION TITLE
Use async await inside subscribe block

### DISCUSSION THREAD

**Discussion by prabh-62** [ReactiveX/rxjs#6852] [General]
Is it possible to use async await inside `subscribe` block?
<img width="345" alt="rxjs_observable_async_await" src="https://user-images.githubusercontent.com/13006107/156200494-8445f97c-b6ac-47d8-a18d-05478503a4e5.png">

**Comment by benlesh** [comment]
Yes, it seems that something is strange with your TypeScript configuration?

[Example of it working here](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBAbzhAZgGjiAhmDB5AIwGcBTKANywIBsS4BfOFKCEOAcigA8ArI9gNwBYAFCiUAVwB2AYxjAIUuEVokwAChBEAXHCkSQBMgEpEouHCgkYEqEqkkA7nAAKLEMFLqrRCNXJ0ALwAfMrWACqgJBASMN4kvv4kGFrGxsIi9KKiqOrsAOrQ1AAm7MbmcAB0YMBgJJo46upSWCAkpiFwAAYAEiTU1BAYACQILW30AIRdaRWVRBLEMlDARupYRACesnCU1BJBoQgVFliOWMDwKiRq6gCMAAxP6SdwMoqJJJWDAObqewcXiILPQgUA)

**Comment by prabh-62** [comment]
Typescript doesn't complain, I can compile the code. However, ESLint does complain
- [@typescript-eslint/no-misused-promises](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-misused-promises.md)

**Reply by benlesh** [reply]
Yeah. I guess I just disagree with that lint rule. It's not misused. If it didn't compile under TypeScript, that would be one thing, but I don't think we can accommodate everyone's lint rules.
